completions/bash/swaygrab

```diff
@@ -0,0 +1,48 @@
+# swaygrab(1) completion
```

Comment:
`swaygrab` has been removed. We should remove completions too.

Reply:
OK, sure.
I haven't followed the development of sway; why was `swaygrab` removed? Is its functionality available some other way?

Reply:
Yeah, you can now use tools like [grim](https://github.com/emersion/grim).

Reply:
If possible, can you open a PR to remove all `swaygrab` references in docs and completion?

```
meson.build
185:		'completions/zsh/_swaygrab',

sway/sway.1.scd
95:*sway*(5) *swaymsg*(1) *swaygrab*(1) *sway-input*(5) *sway-bar*(5)

swayidle/swayidle.1.scd
61:*sway*(5) *swaymsg*(1) *swaygrab*(1) *sway-input*(5) *sway-bar*(5)

completions/zsh/_swaygrab
1:#compdef swaygrab
6:# Completion script for swaygrab in sway wm (http://swaywm.org)
```